scipy/stats/_unuran/unuran_wrapper.pyx.templ

```diff
@@ -223,7 +224,7 @@ cdef object _setup_unuran():
 _setup_unuran()
 
 
-cdef dict _unpack_dist(object dist, str dist_type, list meths,
+cdef dict _unpack_dist(object dist, str dist_type, list meths = None,
```

Comment:
Just to check my understand, this and `if meths is not None` below is not strictly necessary with this PR, but improving something else?

Reply:
I think it is to allow specifying either `pdf` or `logpdf`. Currently, `meths` is the list of methods that must be present but since `pdf` and `logpdf` are conditionally optional, I think it makes sense to check outside of this function id the required methods are present.

Reply:
I guess one could also pass an empty list and drop the "if .. is not None". I followed the same approach as for the optional methods for consistency